services/common/src/components/documents/ReplaceDocumentModal.tsx

```diff
@@ -153,4 +153,14 @@ const mapDispatchToProps = (dispatch) =>
     dispatch
   );
 
-export default connect(null, mapDispatchToProps)(ReplaceDocumentModal);
+export default compose(
```

Comment:
I was thinking we could just use the FormWrapper here?
I don't think we can specify the form as FORM.ADD_EDIT_PROJECT_SUMMARY though, as it won't always be that form. Does it work to specify a new one like FORM.REPLACE_DOCUMENT?